packages/orchestrator-ui-components/src/components/WFOSubscription/utils/utils.tsx

```diff
@@ -1,38 +1,71 @@
-import { EuiIcon } from '@elastic/eui';
 import React from 'react';
+import { EuiIcon } from '@elastic/eui';
 
-export function getColor(num: number) {
-    if (num === 1) return 'warning';
-    if (num === 2) return 'primary';
-    return 'danger';
-}
+import { FieldValue } from '../../../types';
 
-export const GENERAL_TAB = 'general-id';
-export const SERVICE_CONFIGURATION_TAB = 'service-configuration-id';
-export const PROCESSES_TAB = 'processes-id';
-export const RELATED_SUBSCRIPTIONS_TAB = 'related-subscriptions-id';
+const MAX_LABEL_LENGTH = 45;
+
+export enum SubscriptionTabIds {
+    GENERAL_TAB = 'general-id',
+    SERVICE_CONFIGURATION_TAB = 'service-configuration-id',
+    PROCESSES_TAB = 'processes-id',
+    RELATED_SUBSCRIPTIONS_TAB = 'related-subscriptions-id',
+}
 
 export const tabs = [
     {
-        id: GENERAL_TAB,
+        id: SubscriptionTabIds.GENERAL_TAB,
         translationKey: 'tabs.general',
         prepend: <EuiIcon type="devToolsApp" />,
         append: <></>,
     },
     {
-        id: SERVICE_CONFIGURATION_TAB,
+        id: SubscriptionTabIds.SERVICE_CONFIGURATION_TAB,
         translationKey: 'tabs.serviceConfiguration',
         prepend: <EuiIcon type="submodule" />,
     },
     {
-        id: PROCESSES_TAB,
+        id: SubscriptionTabIds.PROCESSES_TAB,
         translationKey: 'tabs.processes',
         prepend: <EuiIcon type="indexRuntime" />,
     },
     {
-        id: RELATED_SUBSCRIPTIONS_TAB,
+        id: SubscriptionTabIds.RELATED_SUBSCRIPTIONS_TAB,
         disabled: true,
         translationKey: 'tabs.relatedSubscriptions',
         prepend: <EuiIcon type="heatmap" />,
     },
 ];
+
+export function getColor(num: number) {
```

Comment:
Without loading state : is this till used somewhere?

Reply:
Good point. Will check and remove otherwise

Reply:
It's not. I removed it